packages/auth/src/features/authenticateSuperUser/handler.ts

```diff
@@ -36,9 +38,16 @@ export default async function authenticateSuperUserHandler(
     throw unauthorized()
   }
 
+  if (result.status === 'deactivated') {
```

Comment:
This is just to improve error messaging when you try to seed again after seeding once